solr/conf/solrconfig.xml

```diff
@@ -20,11 +20,25 @@
      this file, see http://wiki.apache.org/solr/SolrConfigXml.
 -->
 <config>
-  <luceneMatchVersion>4.7</luceneMatchVersion>
-  <dataDir>${solr.data.dir:}</dataDir>
+  
+  <!-- Controls what version of Lucene various components of Solr
+       adhere to.  Generally, you want to use the latest version to
+       get all bug fixes and improvements. It is highly recommended
+       that you fully re-index after changing this setting as it can
+       affect both how text is indexed and queried.
+  -->
+  <luceneMatchVersion>5.0.0</luceneMatchVersion>
```

Comment:
As far as I know, there isn't any Solr6 specific configuration here. This will work on version 5 as well as 6.